fe/fe-core/src/main/java/org/apache/doris/cloud/transaction/CloudGlobalTransactionMgr.java

```diff
@@ -1213,6 +1293,26 @@ public boolean commitAndPublishTransaction(DatabaseIf db, long transactionId,
             }
             throw e;
         } finally {
+            stopWatch.stop();
+            String detailMsg = "";
+            if (commitCostTimeStatisticMap.containsKey(transactionId)) {
```

Comment:
can move this into the `if (costTimeMs > 1000)` to skip construct the `detailMsg` if log is not needed

Reply:
done